cmake/modules/RootMacros.cmake

```diff
@@ -1366,9 +1368,18 @@ function(ROOT_ADD_TEST test)
   set(_command ${_command} -DSYS=${ROOTSYS})
 
   #- Handle ENVIRONMENT argument
+  if(asan)
```

Comment:
I would prefer not to add options in ROOT macros, it breaks them when somebody is trying to use them outside of ROOT. Can we use a trick with a check of env variable $ENV{LSAN_OPTIONS} here as well?

Reply:
I don't understand the problem yet. Unfortunately, one has to hack the environment. What would you do instead of hacking it here?

Reply:
I don't understand the problem yet. Unfortunately, one has to hack the environment. What would you do instead of hacking it here?